src/boilerplates/boilerplates.ts

```diff
@@ -93,6 +94,33 @@ export function makeCommandBoilerplates(program: Command) {
             createEmbeddedWalletNextjs(program, options);
         });
 
+        comm
+            .command("create-embedded-wallet-linking-starter")
+            .description("Clone a starter boilerplate for Sequence Embedded Wallet Linking demo integrated with React")
+            .option(
+                "--waas-config-key <waas_key>",
+                "WaaS config key for this project"
+            )
+            .option(
+                "--project-access-key <access_key>",
+                "Project access key for Sequence requests"
+            )
+            .option(
+                "--google-client-id <google_client_id>",
+                "Google client ID to be used during authentication"
+            )
+            .option(
+                "--wallet-connect-id <wallet_connect_id>",
+                "Wallet Connect ID to be used during authentication"
+            )
+            .option(
+                "--verbose",
+                "Show additional information in the output"
+            )
+            .action((options) => {
+                createWalletLinkingEmbeddedWalletReact(program, options);
+            })
+        
```

Comment:
This has too much indentation. It should be aligned with the others.

Reply:
sorry i disagree, we should move all indententation to how this is going about it. typically you leave indentation for then there's a dereferencer

Reply:
nvm i know what you mean